arch/cheerp/bits/alltypes.h.in

```diff
@@ -49,8 +49,6 @@ TYPEDEF _Int64 time_t;
 STRUCT timespec { time_t tv_sec; long tv_nsec; };
 STRUCT __IO_FILE  ;
 
-// #include <__stddef_max_align_t.h>
```

Comment:
Let's do this properly, squash these 2 commits using git rebase.

Also, the debian/rules and every other place (including docs) mentioning CPPFLAGS should be fixed